### DISCUSSION TITLE
How to fix  slide for no-swiping ?

### DISCUSSION THREAD

**Discussion by hu-milize** [nolimits4web/swiper#5483] [Show and tell]
who can tell me how to fix  slide for no-swiping ?
thank you.

**Comment by Arhell** [comment]
describe the problem more precisely
Sandbox demo as an example

upd
Maybe I understood you correctly, try
https://swiperjs.com/swiper-api#parameters

_Set to false to disable swiping to next slide direction (to right or bottom)_
